src/Yoonit.Camera.common.ts

```diff
@@ -281,6 +285,8 @@ export abstract class CameraBase extends ContentView implements CameraDefinition
     public setFaceContours(enable: boolean) {}
 
     public setFaceContoursColor(color) {}
+
+    public setComputerVision(enable: boolean) {}
```

Comment:
Missing the `setComputerVisionLoadModels` function in the `src/Yoonit.Camera.common.ts`.

Reply:
It's only implemented on android and is not used as a prop setter. Should we create an empty function in these cases too?

Reply:
Oh, i think I got it, now linter is warning that camera.common does not implement this method D: